packages/types/src/tokens.ts

```diff
@@ -91,12 +91,19 @@ export interface TokenDataTypes {
   containerNames: string
 }
 
+export type DefaultTokenCategory = keyof TokenDataTypes
+export type TokenCategory = DefaultTokenCategory | (string & {})
+
+type UnknownTokenTypeValue = string | number | Record<string, any>
```

Comment:
```suggestion
type UnknownTokenTypeValue = string | number
```

Reply:
I initially added this & Array<any> cause token types like `borders` / `gradients` / `shadows` allow those, should we ignore those use-cases ?

Reply:
We'd definitely need it for the inbuilt tokens. The idea was to not allow it for generic tokens. If generic tokens accept `Record` there'd have to be a way to transform them to their string value. E.g. An object for custom token `dropShadow` would need to be transformed just like we transform `shadows` internally.

Reply:
oh, makes sense, thanks ! 🙏